vending-app/src/main/java/org/microg/vending/billing/InAppBillingServiceImpl.kt

```diff
@@ -232,7 +232,8 @@ class InAppBillingServiceImpl(private val context: Context) : IInAppBillingServi
         developerPayload: String?
     ): Bundle {
         if (Log.isLoggable(TAG, Log.DEBUG)) Log.d(TAG, "getBuyIntent(apiVersion=$apiVersion, packageName=$packageName, sku=$sku, type=$type, developerPayload=$developerPayload)")
-        return resultBundle(BillingResponseCode.BILLING_UNAVAILABLE, "Not yet implemented")
+        return runCatching { getBuyIntentExtraParams(apiVersion, packageName!!, sku!!, type!!, developerPayload, null) }
```

Comment:
Why is it necessary to catch exceptions here? Shouldn't they be catched in `getBuyIntentExtraParams` if that was necessary?

Reply:
It is not certain whether the passed parameter is empty. Temporarily catch the exception here to avoid abnormal crash.